kolibri/plugins/setup_wizard/assets/src/state/actions/forms.js

```diff
@@ -0,0 +1,21 @@
+function submitDefaultLanguage(store, language) {
+  store.dispatch('SET_LANGUAGE', language);
+}
+function submitFacilityName(store, facilityName) {
+  store.dispatch('SET_FACILITY_NAME', facilityName);
+}
+function submitSuperuserCredentials(store, name, username, password) {
+  store.dispatch('SET_SU_NAME', name);
```

Comment:
Seems odd to make the mutations so granular, when they are never used separately.

Can move the granularity inside the mutation if you want to maintain this explicitness.

Another option if you want to avoid lots of parameters is to use parameter destructuring:

`function submitSuperuserCredentials(store, { name, username, password }) {`